package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "dependencies": {
+    "babel-polyfill": "^6.26.0",
+    "chart.js": "^2.9.3",
+    "jquery": "^3.5.0",
+    "jquery.dirtyforms": "^2.0.0",
+    "lightbox2": "^2.9.0",
+    "semantic-ui-css": "^2.2.0",
+    "slick-carousel": "^1.8.1"
+  },
```

Comment:
I'm not particularly confortable to manage these extensions, what happened if a security issue is raised on one of these dependencies ?
Another point for using submodules, or at least using packages produced by editors